spyder_kernels/customize/spydercustomize.py

```diff
@@ -400,6 +400,8 @@ def transform_cell(code):
 
 def exec_code(code, filename, ns_globals, ns_locals=None, post_mortem=False):
     """Execute code and display any exception."""
+    # Tell IPython to hide this frame
```

Comment:
Could be worthy to add here (and the other places below where the hiding is set) a reference to the IPython version that starts supporting this?